### DISCUSSION TITLE
Markdown issue after building theme CSS from source

### DISCUSSION THREAD

**Discussion by bob3x255** [jpanther/congo#481] [Q&A] [answered]
Hi! 

I am [building the theme css from source](https://jpanther.github.io/congo/docs/advanced-customisation/#building-the-theme-css-from-source) and encountered an issue with markdown styling. 
As far as I understand, Tailwind default styles are not configured for the markdown, e.g., default value for lists is `list-none` and so on. 
Essentially, what happens is that I have a front matter variable or content with following values.
```
1. List item
2. List item
3. List item
```
And it is rendered as `list-none`
![image](https://user-images.githubusercontent.com/28487554/221151405-003c1870-6568-4a82-b34a-4d014fa707b0.png)

It's my first advanced customization of the Hugo theme and I am not quite sure where to dig to be able to add content either using `emojify` or `markdownify` with an appropriate styling. Any suggestion where to look? 

Thanks!

**Comment by bob3x255** [comment, marked as answer]
Figured it out, apparently typography's prose plug-in haven't been applied in my layout. 
I find out the same issue with other SSGs, the solution is to make sure you include `prose` property in layout element.

However, now Markdown formatting is not applied, for instance `<sup>` tags. 

front matter   
```  
content: |
        1. List item
        2. List item
        3. List item

        A<sub>2</sub>
```
layout
```
<div class = "prose mt-6 pb-6 text-lg">
     {{ .content | markdownify  }}
</div>
```
output
<img width="389" alt="image" src="https://user-images.githubusercontent.com/28487554/221175565-4ae2c88a-a330-47a5-9cce-3cee35fbdf79.png">

**Reply by jmooring** [reply]
If you look at the HTML generated from your markdown in Hugo, you will see:

```
<p>A<!-- raw HTML omitted -->2<!-- raw HTML omitted -->
```

Hugo renders markdown to HTML using [Goldmark](https://github.com/yuin/goldmark). 

> By default, goldmark does not render raw HTML or potentially dangerous links. 

See <https://github.com/yuin/goldmark#html-renderer-options>

To override this, place the following in your site configuration file (config.toml):

```text
[markup.goldmark.renderer]
unsafe = true
```

This is not unsafe if _you_ control the content.

If you look at this theme's example site, you can see that this has already been done for you, if you use the example configuration:
<https://github.com/jpanther/congo/blob/dev/exampleSite/config/_default/markup.toml>

**Reply by bob3x255** [reply]
Thanks! I literally looked everywhere except this one.
